pickhardtpayments/OracleLightningNetwork.py

```diff
@@ -71,3 +71,22 @@ def theoretical_maximum_payable_amount(self, source: str, destination: str, base
 
         mincut, _ = nx.minimum_cut(test_network, source, destination)
         return mincut
+
+    def settle_payment(self, path: OracleChannel, payment_amount: int):
+        """
+        receives a channel and a payment amount and adjusts the balances of the channels along the path.
+
+        settle_payment should only be called after send_onion terminated successfully!
+        There is currently no further exception handling if along the path channels do not provide
+        enough liquidity!
+        # TODO testing
+        """
+        for channel in path:
+            settlement_channel = self.get_channel(channel.src, channel.dest, channel.short_channel_id)
```

Comment:
we will also need to add the sats to the channel that goes in the other direction. so not only removing liquidity form the sender but also adding it to the destination. so there will be two channels to work with.

Reply:
fix in 4ecadee